ui-e2e/scripts/deploy.js

```diff
@@ -0,0 +1,22 @@
+const path = require('path')
+const shell = require('shelljs')
+
+const envsDir = path.join(__dirname, '../envs')
+const deployContractsDir = path.join(__dirname, '../../contracts-2.1.0/deploy')
```

Comment:
does it make sense to introduce a module with the constant to keep the name of the directory containing the bridge contracts as so this module and this constant will be used in another file `ui-e2e/scripts/deployERC20.js`. So, later when the directory with contracts is renamed it will be necessary to change just one file.

Reply:
Done.